falcon/routing/converters.py

```diff
@@ -132,8 +152,22 @@ def convert(self, value):
             return None
 
 
+class PathConverter(BaseConverter):
+    """Field converted used to match the rest of the path.
+
+    This field converter matches the all the remaining url path,
```

Comment:
`matches the all the remaining url path` -->  `matches the remainder of the URL path`